test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -1039,6 +1040,14 @@ func TestProjectWithStreamsProcessing(t *testing.T) {
 									),
 									Namespace: targetNamespace,
 								},
+								// sample_stream_solar is a sample connection which is added by default to new stream instances
+								{
+									Name: resources.NormalizeAtlasName(
+										fmt.Sprintf("%s-%s-%s", generator.projectName, generator.streamInstanceName, sampleStreamName),
```

Comment:
From what I can tell, doing this doesn't achieve what we want in tests, because the generated project names are so long that the instance and connection names are truncated (so both of these have the same name in k8s). This only affects tests, but I don't know if we want to address it.